File: src/codes/Users_Manual/environment.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using var autd = Controller.Open([new AUTD3()], new Nop());
autd.Environment.SoundSpeed = 340e3f;
autd.Environment.SetSoundSpeedFromTemp(15);
var wavelength = autd.Environment.Wavelength();
var wavenumber = autd.Environment.Wavenumber();

File: src/codes/Users_Manual/environment.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::open([AUTD3::default()], autd3::link::Nop::new())?;
autd.environment.sound_speed = 340e3;
autd.environment.set_sound_speed_from_temp(15.);
let wavelength = autd.environment.wavelength();
let wavenumber = autd.environment.wavenumber();
# Ok(())
# }